cappy.py

```diff
@@ -123,11 +138,12 @@ def handle_inbound_request(self, message):
             request_id = message['id']
             args = message['args']
         except KeyError as e:
+            print(e)
             raise  # TODO actually generate an erro message instead
 
         result = getattr(self.implementation, method_name)(*args)
         response = {
-                "id": request_id,
-                "result": result
+            "id": request_id,
+            "result": result
```

Comment:
I'm not entirely sure, the formatting changes were mostly done by [autopep8](https://pypi.python.org/pypi/autopep8)